### ISSUE TITLE
Report enteric methane directly and ONLY from animal module

### ISSUE THREAD

**Issue by tomhuhh** [RuminantFarmSystems/RuFaS#698]
**Is your feature request related to a problem? Please describe.**
Due to the current set up of the code structure, enteric methane emission is passed on to the manure module together with the manure excretions. However, there is no need to pass the enteric methane estimates to the manure module. 

**Describe the solution you'd like**
The goal should be to report the enteric methane directly and ONLY from the animal module. 

**Describe alternatives you've considered**
NA

**Additional context**
This is a request beyond version 1.

**Comment by tomhuhh** [comment]
@ltp23 Hello Loi, I tentatively assigned you to this issue because I know you have been working on some of the stuff. However, feel free to let me know who you think might be a better option to be assigned with this issue. Also please keep in mind that this issue is beyond version 1. Thank you!

**Comment by PooyaHekmati** [comment]
TODO: Follow up with author to check if this is still the case.

**Comment by tomhuhh** [comment]
I believe that this issue has not been resolved yet. However, I also think this is at a lower priority because the current way of reporting enteric methane works fine. This could potentially be taken into consideration during the animal module refactoring led by @JoeWaddell. Thanks!

**Comment by ew3361zh** [comment]
- This should be addressed after PR #1737 gets merged.
- Check in on status of #657 as well.

**Comment by Isgotkowitz** [comment]
Blocked by #1680

**Comment by allisterakun** [comment]
This is will be taken care of in issue #1927
